torchdynamo/output_graph.py

```diff
@@ -279,6 +281,7 @@ def compile_subgraph(self, tx, partial_convert=False):
             and self.side_effects.is_empty()
         ):
             # optimization to generate better code in a common case
+            # TODO(voz): Add spec here too before land
```

Comment:
Self review: This will be added before land, if the rest of the direciton looks fine